frontend/src/Onboarding/GenericOnboarding.tsx

```diff
@@ -113,7 +115,12 @@ const theme = createMuiTheme({
   },
 });
 
-const steps = ["", "", "", ""];
+const steps = [
+  "Student Profile",
+  "Completed Courses",
+  "Transfer Courses",
+  "AP/IB Courses",
+];
```

Comment:
added steps so that users can view what steps they have completed/remaining. we can totally change the text here. changed this due to a warning, see below